app/controllers/api/v1/auth_sessions_controller.rb

```diff
@@ -1,41 +1,176 @@
 class Api::V1::AuthSessionsController < Devise::SessionsController
+  include RackSessionsFix
+  require 'jwt'
+
+  respond_to :json
+
   protect_from_forgery with: :null_session
-  before_action :sign_in_params, only: :create
-  before_action :load_user, only: :create
+  #before_action :sign_in_params, only: :create
+  #before_action :load_user, only: :create
+  #skip_before_action :verify_authenticity_token, :only => :create
```

Comment:
We can remove lines 8 & 9 if we're no longer using them. I would like to remove 10 as well if you're not planning on using it in the near future. I think that things you'd like to save for reference later would be good to save directly on the pull request. You can always come back and reference it later.